thirdeye/thirdeye-frontend/app/helpers/manage-alert-utils.js

```diff
@@ -0,0 +1,226 @@
+import { helper } from '@ember/component/helper';
+import moment from 'moment';
+
+/**
+ * Handles types and defaults returned from eval/projected endpoints
+ * @param {Number|String} metric - number or string like 'NaN', 'Infinity'
+ * @param {Boolean} allowDecimal - should this metric be shown as whole number?
+ * @returns {Object}
+ */
+export function formatEvalMetric(metric, allowDecimal = false) {
+  let shown = 'N/A';
+  if (Ember.typeOf(metric) === 'number') {
+    if (allowDecimal) {
+      shown = (Number(metric) === 1 || Number(metric) === 0)
+        ? metric * 100
+        : (metric * 100).toFixed(1);
+    } else {
+      shown = metric;
```

Comment:
Can metric ever be a string form of a number? If so, should you apply the allow decimal logic to that as well?

Reply:
Sure - I'll use !isNaN instead here.